go/protocol/keybase1/extras.go

```diff
@@ -644,11 +645,35 @@ func (s SigID) Equal(t SigID) bool {
 	return s == t
 }
 
-func (s SigID) EqualIgnoreLastByte(t SigID) bool {
-	if len(s) != len(t) || len(s) < 2 {
-		return false
+func (s SigID) EqualTrimSuffix(t SigID) bool {
+	sTrimmed := s.TrimSuffix()
+	tTrimmed := t.TrimSuffix()
+	return !sTrimmed.IsNil() && !tTrimmed.IsNil() && sTrimmed.Equal(tTrimmed)
```

Comment:
heh when you put it this way seems like nil should equal nil

Reply:
yeah, but i was erring on the side of not equal.  Alternatively we can plumb errors through, which might actually be OK since only one caller. What do you think?